internal/pkg/levee/levee.go

```diff
@@ -69,15 +77,32 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
+func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, suppressedNodes suppression.ResultType, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
```

Comment:
This could be work for a future PR, but it could be worth printing or emitting a `Diagnostic` if we hit a point where suppression was not needed.

Reply:
Agreed! (This possibility is in fact mentioned in the PR description.)